client/domain/SnapshotObjectFactorySpec.js

```diff
@@ -30,6 +30,13 @@ describe('SnapshotObjectFactory', function() {
     });
   }));
 
+  describe('setPrereqCheckFailure', function() {
+    it('should correctly set and get prereqCheckFailure', function() {
+      snapshot.setPrereqCheckFailure("true");
```

Comment:
Is this an actual possible value for PrereqCheckFailure that can be achieved in practice?